packages/fx-core/templates/plugins/resource/spfx/solution/README.md

```diff
@@ -15,7 +15,7 @@ The SharePoint Framework (SPFx) is a page and web part model that provides full
 
 ## Prerequisites
 
-> - Node.js v16
+- [Node.js](https://nodejs.org/en/), supported versions: 16
```

Comment:
Please also help update README in SPFx template under `src` folder.

Reply:
Updated, thx.

Reply:
Maybe it is for V3. Here we only change v2 templates